### PR TITLE
Mobile first and mobile responsive css 

### PR DESCRIPTION
**Pull request by jennah2121**
The css is almost fully mobile responsive excluding the help button which overlaps slightly on some occasions. 

dom.js 
- some of the class names were change to have a '-js' added to them so that its easier to distinguish between classes used for styling and those used for js dom manipulation 

hbs files 
- All have had a section added to them to make them more semantic 
- class names changed to be more bem

profile.hbs
- This is the file with the most significant change
- The html has been restructured/ rewritten to allow for easier styling 

Note: 
The client side validation does not seem to be working however, this seems to have been a problem prior to this PR

### CODE REVIEW

**Comment by eadehemingway** [comment]
Jennah this looks great, can I just check - does it still look good on desktop or have you prioritised the mobile view?

**Comment by jennah2121** [comment]
Thanks :) Mobile view has been prioritised but it still looks good on larger screens. Pull the branch and have a look to see what you think...

File: src/views/profile.hbs [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-    <img src="/assets/smallGraph.png" alt="">
-    <br> See Data</a>
-</div>
\ No newline at end of file
+<section class="profile--section">
+  <h1 class="title">profile</h1>
+  <div class="profile--options">
+    <a class="profile--links" href="/symptoms/home">
+      <div class="profile--container-img">
+        <img src="/assets/redcross.png" alt="A plus icon">
```
Comment (tspeed90) [inline src/views/profile.hbs]: Hey Jennah, this page looks a ton nicer already. One thing, I'm not sure we want alt text on those icons. If anything, they should maybe be aria-hidden? I think they are more for decorative use so maybe we don't want them read aloud. Thoughts?

Comment (jennah2121) [reply src/views/profile.hbs]: I think you're right Tammy. I will remove the alt text and use an empty alt attribute instead. This link ([webAIM Alternate Text](https://webaim.org/techniques/alttext/) ) seems to suggest that an empty alt without aria-hidden is okay.

Comment (tspeed90) [reply src/views/profile.hbs]: Sounds perfect! Thanks. 

New problem: Travis failed for some reason.

Comment (jennah2121) [reply src/views/profile.hbs]: That's so strange... Travis seems to be passing now

**Comment by tspeed90** [comment]
Hey Jennah, I just pulled the branch to check it out on mine and the nav bar font seem really small now on mine. I know sometimes what it looks like on a Mac and what it looks like on a Windows/Linux machine can be very different, so I've included a screen shot to show you what it looks like for me. 

correl_screenshot

**Comment by tspeed90** [comment]
Thank you Jennah for making those changes! Am I safe to merge it if I approve?

**Comment by jennah2121** [comment]
You're welcome. I think we're just waiting on Eade but then you can merge

**Comment by tspeed90** [comment]
:+1: Sounds good. I've just made a small change too that I noticed is still a problem when I was looking through these changes. I'll just wait for this to be merged and then pull it and make my PR.  :smiley:

**Review by tspeed90 — APPROVED** [review]

**Comment by eadehemingway** [comment]
there seem to be some stylistic changes that i think looked better before... 
the two main ones being:
- the rectangle profile buttons not being square anymore
- there not being the grey panels framing the content on desktop view

if i am outvoted and everyone else likes it better like this thats fine, but just thought I would voice it

**Comment by jennah2121** [comment]
Hey @eadehemingway thanks for the review!! 

For me personally, I think that the stylistic changes are minor and don't impact significantly on the overall feel of the app, particularly as the grey frames aren't in the original Figma designs. However,  I'm more than happy for you to pull the branch and make your stylistic changes. Alternatively if everyone agrees to merge you could make a separate PR to make those changes. Unfortunately at the moment I have another project with a pressing deadline which has to take priority over Correl.

Overall, given that the changes made in this PR are significant in terms of fixing bugs (namely the fact that correl can't be used on mobile at all) I think at this point it should be merged. I do appreciate your comments though! 🙂

Thoughts??

**Comment by tspeed90** [comment]
I pulled the branch and fixed the squares on the profile page because I agree that they looked a bit odd after they become more rectangular. 

I will try to work on the grey panels. I played around with it for a moment and didn't get it working but mostly because I haven't been one of the main people working in the css, so I don't quite know where everything is and how it all works. 

I looked at the site in mobile and it looks SO much better. Well done @jennah2121 and @eadehemingway!

**Pull request closed without merging**